app/views/director/kids/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1> Add a Kid to Camp! </h1>
+
+<%= form_for @kid do |form| %>
+
+<%= form.label :first_name %>
+<%= form.text_field :first_name %>
+
+<%= form.label :last_name %>
+<%= form.text_field :last_name %>
+
+<%= form.label :birthdate %>
+<%= form.date_field :birthdate %>
+
+<%=form.label :social_insurance_number %>
```

Comment:
Same thing over here. It's the `<%= form.text_field :sin %>` that tells the form what key the social insurance number should have within the params.